### DISCUSSION TITLE
WSA not launching into Play Sotre and Magisk

### DISCUSSION THREAD

**Discussion by Wlz-23** [LSPosed/MagiskOnWSALocal#372] [Q&A]
My WSA instances seem to just hang forever and then after a while Windows kills them. I saw AlexanderSLoburev's post with a similar issue and Soetsu09's solution. It has not worked for me and I continue to be stuck in a boot loop of sorts.

Anyone else have some steps I can take to solve my issue?

![image](https://user-images.githubusercontent.com/108379677/215274426-1f2e4eb4-1d26-4770-a26f-671da37791af.png)

**Comment by Almighty188** [comment]
Yes, I had the problem too and managed to fix it on my own, see the other thread.  Basically, can you share how you built the WSA?  Did you use ./run.sh or did you use ./build.sh?  What options did you use?  The next thing is open up Windows Subsystem for Android Settings and then select the Compatibility tab, does it show any apps installed?  Post a screenshot of that as this is what mines looks like but mines only starts Play Store and not Magisk even though it's installed.

![image](https://user-images.githubusercontent.com/78993393/224493203-f5893a5a-8974-4de8-baa7-bc793b3544bb.png)

**Reply by lazytiger** [reply]
I have the same problem, and my installed apps are empty. I just use build.sh without any other parameters. How do I fix this?

**Reply by Almighty188** [reply]
Does using run.sh work?
